packages/react/src/components/dateInput/DateInput.test.tsx

```diff
@@ -14,9 +14,13 @@ describe('<DateInput /> spec', () => {
   const RealDate = Date;
   const mockDate = (isoDate: string) => {
     (global as any).Date = class extends RealDate {
-      constructor(...theArgs: [number, number, number, number, number, number]) {
```

Comment:
Eslint didn't like this for some reason. Cannot read map of undefined it complained

Reply:
I was thinking is it because there might not be any arguments at some point? Would default value `theArgs = []` help?

Reply:
Tried this but could not get it working

Reply:
Was able to fix this by updating typescript-eslint plugin, as it was incompatible with the updated version of typescript